backend/public-graph/graph/resolver.go

```diff
@@ -2384,7 +2384,10 @@ func (r *Resolver) AddSessionEvents(ctx context.Context, sessionID int, events *
 
 				attributes := make(map[string]string)
 				for k, v := range propertiesObject {
-					attributes[k] = fmt.Sprintf("%v", v)
+					attributes[k] = fmt.Sprintf("%.*v", SESSION_FIELD_MAX_LENGTH, v)
+					if len(attributes[k]) > 0 {
+						fields[k] = attributes[k]
```

Comment:
Are we concerned with collisions in the fields object? What if multiple click events write to fields with a `clickSelector`? I believe we will only get the last one

Reply:
good point, guess that's a general problem with this function? fields should be an array rather than a map since we allow multiple values for the same field